packages/mask/src/plugins/RedPacket/SNSAdaptor/SelectNftTokenDialog.tsx

```diff
@@ -260,19 +429,52 @@ export function SelectNftTokenDialog(props: SelectNftTokenDialogProps) {
         setSearched(false)
     }, [tokenId])
 
+    useEffect(() => {
+        if (tokenDetailedOwnerList.length > 0) setTokenDetailed(undefined)
+    }, [tokenDetailedOwnerList])
```

Comment:
Here we only care about `tokenDetailedOwnerList.length`

Might reduce re-rendering with:
```suggestion
    }, [tokenDetailedOwnerList.length])
```